ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java

```diff
@@ -254,26 +250,7 @@ private void createBanner() {
         listener.onAdFailedToLoad(AdRequest.ERROR_CODE_INTERNAL_ERROR);
       }
     } else {
-      View adView = null;
-      VungleNativeAd vungleMRECBanner = Vungle.getNativeAd(placementId, mAdConfig, playAdCallback);
-      if (vungleMRECBanner != null) {
-        adView = vungleMRECBanner.renderNativeView();
-      }
-
-      if (adView != null) {
-        Log.d(TAG, "display MREC:" + vungleMRECBanner.hashCode() + this);
-        if (vungleBannerAd != null) {
-          vungleBannerAd.setVungleMRECBanner(vungleMRECBanner);
-        }
-
-        updateVisibility(mVisibility);
-        adView.setLayoutParams(adParams);
-        // don't add to parent here
-        listener.onAdAvailable();
-      } else {
-        // missing resources
-        listener.onAdFailedToLoad(AdRequest.ERROR_CODE_INTERNAL_ERROR);
-      }
+      listener.onAdFailedToLoad(AdRequest.ERROR_CODE_INTERNAL_ERROR);
```

Comment:
minor: we can remove this block or propose to use error code `AdRequest.ERROR_CODE_INVALID_REQUEST`
refer to: https://github.com/Vungle/googleads-mobile-android-mediation/blob/ccfd2c6cdccc5a6409482c81a0626bcbb97cc3a3/ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java#L247-L253

Reply:
Use ERROR_CODE_INVALID_REQUEST.